rpm/xcache.spec

```diff
@@ -172,6 +172,7 @@ mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
 %config %{_sysconfdir}/xrootd/config.d/40-stash-cache-plugin.cfg
 %config %{_sysconfdir}/xrootd/config.d/50-stash-cache-authz.cfg
 %config %{_sysconfdir}/xrootd/config.d/50-stash-cache-paths.cfg
+%config(noreplace) %{_sysconfdir}/xrootd/config.d/90-stash-cache-disks.cfg
```

Comment:
where are the lines for .../90-atlas-xcache-disks.cfg and .../90-cms-xcache-disks.cfg?

Reply:
They're already there. Before this PR, the ATLAS and CMS packages had their own files